iOSClient/Media/NCMediaCommandView.swift

```diff
@@ -67,12 +68,60 @@ class NCMediaCommandView: UIView {
         gradient.frame = bounds
     }
 
-    @IBAction func selectButtonPressed(_ sender: UIButton) {
+    @IBAction func selectcancelButtonPressed(_ sender: UIButton) {
+
+        mediaView.isEditMode = !mediaView.isEditMode
+        setSelectcancelButton()
+    }
+
+    func setSelectcancelButton() {
+
+        mediaView.selectOcId.removeAll()
+        mediaView.tabBarSelect?.selectCount = mediaView.selectOcId.count
+
+        if mediaView.isEditMode {
+            selectcancelButton.setTitle( NSLocalizedString("_cancel_", comment: ""), for: .normal)
+            selectcancelButtonTrailing.constant = 8
+            mediaView.tabBarSelect?.show(animation: true)
+        } else {
+            selectcancelButton.setTitle( NSLocalizedString("_select_", comment: ""), for: .normal)
+            selectcancelButtonTrailing.constant = 46
+            mediaView.tabBarSelect?.hide(animation: true)
+        }
 
-        mediaView.isEditMode = true
         mediaView.collectionView.reloadData()
+    }
+
+    func setTitleDate() {
+
```

Comment:
What's the point of this empty line here? I see it on so many files. Can you change your settings?

Reply:
New functions made by you always have an empty line on the top. Can you change this?